packages/legend-query/src/components/QueryBuilderExplorerPanel.tsx

```diff
@@ -704,6 +705,7 @@ const QueryBuilderExplorerTree = observer(
               queryBuilderState.graphManagerState,
               property,
               node,
+              guaranteeNonNullable(queryBuilderState.querySetupState.mapping),
```

Comment:
Can you make sure we are safe with this NonNullable check. Ideally we don't show a tree if no mapping but lets make sure we don't break the app with this mapping input.